show/muxcable.py

```diff
@@ -907,15 +911,21 @@ def version(port):
             read_side = sonic_y_cable.y_cable.check_read_side(physical_port)
             if logical_key in y_cable_asic_table_keys:
                 if read_side == 1:
-                    get_firmware_dict(physical_port, 1, "self", mux_info_dict)
-                    get_firmware_dict(physical_port, 2, "peer", mux_info_dict)
-                    get_firmware_dict(physical_port, 0, "nic", mux_info_dict)
-                    click.echo("{}".format(json.dumps(mux_info_dict, indent=4)))
+                    get_firmware_dict(physical_port, 1, "self", mux_info_dict, mux_info_active_dict)
+                    get_firmware_dict(physical_port, 2, "peer", mux_info_dict, mux_info_active_dict)
+                    get_firmware_dict(physical_port, 0, "nic", mux_info_dict, mux_info_active_dict)
+                    if active is True:
+                        click.echo("{}".format(json.dumps(mux_info_active_dict, indent=4)))
```

Comment:
It seems like a bit of overkill to add a `mux_info_active_dict` parameter to `get_firmware_dict`. I feel like all of this could be simplified to something like the following:

```python
if active is True:
    mux_info_active_dict = {}
    for key in mux_info_dict:
        if key.endswith("_active"):
            mux_info_active_dict[key] = mux_info_dict[key]
    click.echo("{}".format(json.dumps(mux_info_active_dict, indent=4)))
```

Reply:
fixed